tensorflow/compiler/xla/service/spmd/grad_acc_rewrite.cc

```diff
@@ -33,12 +31,37 @@ HloInstruction* GetAllReduce(HloInstruction* src) {
   return nullptr;
 }
 
+HloInstruction* MaybeBitCast(HloInstruction* src, const Shape& dst_shape) {
+  if (ShapeUtil::SameElementType(src->shape(), dst_shape)) {
+    return src;
+  }
+  const Shape new_shape =
+      ShapeUtil::ChangeElementType(src->shape(), dst_shape.element_type());
+  return src->parent()->AddInstruction(
+      HloInstruction::CreateConvert(new_shape, src));
+}
+
 HloInstruction* MaybeReshape(HloInstruction* src, const Shape& dst_shape) {
```

Comment:
```suggestion
HloInstruction* MaybeReshapeBitCast(HloInstruction* src, const Shape& dst_shape) {
```

Reply:
I use convert to replace bitcast because bitcast is an internal instruction used by layout assignment. So I changed it into `MaybeReshapeConvert` accordingly